README.md

```diff
@@ -145,6 +156,174 @@ readData(dataInput) {
 }
 ```
 
+# Setting Up Client Network
+
+### 7. Enabling Client TLS/SSL
+
+You can use TLS/SSL to secure the connection between the client and members. If you want TLS/SSL enabled
+for the client-cluster connection, you should set an SSL configuration. Once set, the connection (socket) is
+established out of an `options` object supplied by the user. Please see [TLS/SSL section](#1-tlsssl).
+
+As explained in the [TLS/SSL section](#1-tlsssl), Hazelcast members have key stores used to identify themselves (to other members) and Hazelcast Node.js clients have certificate authorities used to define which members they can trust. Starting with Hazelcast 3.8.1 release, mutual authentication is introduced. This allows the Node.js clients also to have their private keys and public certificates and members to have their certificate authorities so that the members can know which clients they can trust. Please see the [Mutual Authentication section](#13-mutual-authentication).
+
+
+# Securing Client Connection
+
+## 1. Supported Security Protocols
+
+This chapter describes the security features of Hazelcast Node.js Client. These features allow you to perform security activities, such as TLS/SSL and its sub-section Mutual Authentication. All of the Security features explained in this chapter are the features of **Hazelcast IMDG Enterprise** edition.
```

Comment:
This chapter describes the security features of Hazelcast Node.js Client. These include using TLS/SSL for connections between clients and members (*between clients too?*) and mutual authentication. These security features are the features of **Hazelcast IMDG Enterprise** edition.

Reply:
I would prefer `requires Hazelcast IMDG Enterprise` instead of `are features of Hazelcast IMDG Enterprise`.

Reply:
I will merge these two.